omg/entities/player.py

```diff
@@ -9,8 +9,8 @@
 from omg.mechanics import movement
 from omg.structural.observer import ObservableSprite
 
-MOVEMENT_SPEED_FORWARD = 1
-MOVEMENT_SPEED_SIDE = 1
+MOVEMENT_SPEED_FORWARD = 5
+MOVEMENT_SPEED_SIDE = 5
```

Comment:
Maybe naming these two as the following be better:
MOVEMENT_SPEED_VERTICAL
MOVEMENT_SPEED_HORIZONTAL

Reply:
agreed, done